Sources/MapboxNavigation/FeedbackViewController.swift

```diff
@@ -156,6 +154,8 @@ public class FeedbackViewController: UIViewController, DismissDraggable, UIGestu
     func commonInit() {
         self.modalPresentationStyle = .custom
         self.transitioningDelegate = self
+
+        currentFeedbackUUID = self.recordCurrentFeedback()
```

Comment:
Is it OK to make a screenshot implicitly in the constructor of this class?

Reply:
Agreed to make currentFeedbackUUID lazy var to avoid creation of cached UUID in the init method.

But there're another topics to discuss:
- in the future we might want to have an opportunity to control the moment of feedback creation explicitly (say, using public API method)
- this also causes creation and immediate removal of feedback screenshot in the `dismissFeedbackItem` method.

Reply:
Reverted, as it is better to call `recordFeedback` in the constructor, otherwise the screenshot will have a different UI at the moment of sending.